src/routes/preHandlers/courseBills.js

```diff
@@ -13,14 +13,12 @@ const { language } = translate;
 
 exports.authorizeCourseBillCreation = async (req) => {
   const { course: courseId, company: companyId, payer } = req.payload;
-  const companyExists = await Company.countDocuments({ _id: companyId }, { limit: 1 });
-  if (!companyExists) throw Boom.notFound();
 
-  const course = await Course.findOne({ _id: courseId }, { type: 1, companies: 1, expectedBillsCount: 1 }).lean();
+  const course = await Course
+    .findOne({ _id: courseId, companies: companyId }, { type: 1, expectedBillsCount: 1 })
```

Comment:
Il ne faut pas ajouter `$in` ? Ca fonctionne comme ca ?

Reply:
le $in n'est pas utile. Il est pertinent quand ce que tu as derrière est un tableau, par exemple tu veux renvoyer les formations qui ont comme structure une structure comprise dans [A, B] (c'est un peu un _ou_). Ici on a qu'une seule valeur possible donc inutile de mettre {$in: [companyId]}